packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js

```diff
@@ -56,7 +56,7 @@ const tests = {
     {
       code: normalizeIndent`
         function MyComponent() {
-          const local = {};
+          const local = useMemo(() => ({}), []);
```

Comment:
Can we just change these tests to be something like `const local = foo()`? To keep them simple and focused on one thing.

Applies to this one and all other tests below that don't specifically check the feature you're adding.

Reply:
https://github.com/facebook/react/pull/19590/commits/eaa2aff9424e7ded164799fa2aef27ccfa4ed3d0